composer.json

```diff
@@ -1,33 +1,35 @@
 {
-    "name": "php-pm/php-pm",
```

Comment:
sorry.. autoreformat in storm :-)

On Aug 30, 2016 15:38, "andig" <notifications@github.com> wrote:

> In composer.json
> <https://github.com/php-pm/php-pm/pull/164#discussion_r76795861>:
>
> > @@ -1,33 +1,35 @@
> >  {
> > -    "name": "php-pm/php-pm",
>
> why reformat the entire file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/php-pm/php-pm/pull/164/files/0c45bf827cfa3bc7e902125ca8a8536b9ddc656a#r76795861>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABqUD7t88BMsnXKhvAmmE4pYTdVcHNb2ks5qlDJbgaJpZM4JwdWg>
> .
>